source/drivers/KeyValueStorage.cpp

```diff
@@ -110,7 +110,7 @@ void KeyValueStorage::scratchKeyValueStore(KeyValueStore store)
   */
 void KeyValueStorage::scratchKeyValuePair(KeyValuePair pair, int scratchOffset)
 {
-    memcpy(this->scratch + scratchOffset, &pair, sizeof(KeyValueStore));
+    memcpy(this->scratch + scratchOffset, &pair, sizeof(KeyValuePair));
```

Comment:
funny.

Reply:
This took me at least 4 hours to track down.

Reply:
😠 Careless 12am code!